app/routes/cart.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    // get the cart our of local storage (this way it persists in the user's browser past a reload)
+    let cart = JSON.parse(localStorage.getItem("cart"));
```

Comment:
having added the cart service, we could now get rid of this and instead have the service handle reading from the local-storage on init + keeping it up to date on updates and the template just read from the service